src/main/webapp/app/shared/monaco-editor/monaco-editor.component.ts

```diff
@@ -289,7 +298,7 @@ export class MonacoEditorComponent implements OnInit, OnDestroy {
     }
 
     changeTheme(artemisTheme: Theme): void {
-        monaco.editor.setTheme(artemisTheme === Theme.DARK ? 'vs-dark' : 'vs-light');
+        this.monacoEditorService.applyTheme(artemisTheme);
```

Comment:
we don't you move this section (with the effect I proposed in the diff editor) directly to the monacoEditorService?

Reply:
Moved the effect to the service 👍